ideas.md

```diff
@@ -29,9 +31,11 @@ The ideas here are all about possible contracts that can interact between public
 
 * Education: a private shard maintains grades and attendance for students, public shard contains different applications that can leverage this information. For example scholarship that is given out on a regular basis based on the grades.
 * Supply chain: across two private shards, passing information about components of the bigger item while maintaining proof of the components origin.
+* Pic ID：a platform for people to record their ownership of some pictures onchain.you can only use the picture taken by yourself but not download other's photo. If some sellers really want to use some photos from others, they must prove that that's the same product and pay to the author.
```

Comment:
What incentivizes people to upload their photos to this app?

Reply:
A nice picture will help sells, so if some sellers take a really nice pic with an expensive camera. He may want to protect his IP.

Reply:
we actually looked into "digiarts" on blockchain, not very promising right now.